devel/geoipupdate/Portfile

```diff
@@ -1,37 +1,138 @@
+                        size    50815 \
+                    github.com/pmezard/go-difflib \
+                        lock    v1.0.0 \
+                        rmd160  fc879bfbdef9e3ff50844def58404e2b5a613ab8 \
+                        sha256  7cd492737641847266115f3060489a67f63581e521a8ec51efbc280c33fc991f \
+                        size    11409 \
+                    github.com/niemeyer/pretty \
+                        lock    a10e7caefd8e \
+                        rmd160  46bcfc3db9e3d98acbacd1f96d9483fa360f88b7 \
+                        sha256  97b952a32175ba84349ef352e523bfa15bf3a06e07e44458a908061fbc519b40 \
+                        size    9405 \
+                    github.com/kr/text \
+                        lock    v0.2.0 \
+                        rmd160  48558c7e8ff67d510f83c66883907e95f4783163 \
+                        sha256  2f2e21ac8a9d523e88cbba4039441defc4a66bfaa78811c900a88fcf28729c4c \
+                        size    8702 \
+                    github.com/gofrs/flock \
+                        lock    v0.8.1 \
+                        rmd160  7ccfc1c50e85c62dab7e3ad2b0c7a0212c4f481c \
+                        sha256  6e35685e7796d4b36b90d4a908270e16868a44e06fcb890c1ec13d34cbb092ea \
+                        size    9059 \
+                    github.com/davecgh/go-spew \
+                        lock    v1.1.1 \
+                        rmd160  7c02883aa81f81aca14e13a27fdca9e7fbc136f7 \
+                        sha256  e85d6afa83e64962e0d63dd4812971eccf2b9b5445eda93f46a4406f0c177d5f \
+                        size    42171 \
+                    github.com/cenkalti/backoff \
+                        lock    v4.2.1 \
+                        rmd160  e4ac343410906a6a1ab0acd578ce034783b5ab42 \
+                        sha256  f30a8b3e79935ada4f425a1e19ba42863bc00e42a981d5b936f7555bd326651d \
+                        size    10399
 
-post-destroot {
-        #- move default conf to GeoIP.conf.default
-        file rename ${destroot}${prefix}/etc/GeoIP.conf ${destroot}${prefix}/etc/GeoIP.conf.default
+build.dir           ${worksrcpath}/cmd/${name}
+build.args          "-ldflags '" \
+                    "-X main.defaultConfigFile=${prefix}/etc/GeoIP.conf" \
+                    "-X main.defaultDatabaseDirectory=${prefix}/share/GeoIP" \
+                    "-X main.version=${version}'"
+
+set docs_path       ${worksrcpath}/doc
+set conf_path       ${worksrcpath}/conf
+
+test.run            yes
+test.cmd            ${build.dir}/${name}
+test.target         {}
+test.args           "-V"
+
+post-build {
+    foreach f [concat [glob -type f -dir ${docs_path} *.md] \
+        ${conf_path}/GeoIP.conf.default] {
+
+        # update paths before generating man pages
+        reinplace -q "s|CONFFILE|${prefix}/etc/GeoIP.conf|g" ${f}
+        reinplace -q "s|DATADIR|${prefix}/share/GeoIP|g" ${f}
+        reinplace -q "s|\$\$||g" ${f}
+    }
+    system -W ${docs_path} "${worksrcpath}/dev-bin/make-man-pages.pl ${docs_path}"
```

Comment:
We may have to both:

1. Ensure this Perl script is using MacPorts Perl
2. Add Perl as a build dependency.

Reply:
I made both changes.